playground/cdk/CdkSample.AppHost/sql.module.bicep

```diff
@@ -0,0 +1,52 @@
+targetScope = 'resourceGroup'
+
+@description('')
+param location string = resourceGroup().location
+
+@description('')
+param principalId string
+
+@description('')
+param principalName string
+
+@description('')
+param principalType string
+
+
+resource sqlServer_l5O9GRsSn 'Microsoft.Sql/servers@2022-08-01-preview' = {
+  name: toLower(take(concat('sql', uniqueString(resourceGroup().id)), 24))
+  location: location
+  properties: {
+    version: '12.0'
+    minimalTlsVersion: '1.2'
+    publicNetworkAccess: 'Enabled'
+    administrators: {
+      administratorType: 'ActiveDirectory'
+      principalType: principalType
+      login: principalName
+      sid: principalId
+      tenantId: subscription().tenantId
```

Comment:
I assume this is equivalent to tenant().tenantId?

Reply:
Yes. We should use `tenant().tenantId`